packages/client/hmi-client/src/components/dataset/tera-dataset-datatable.vue

```diff
@@ -1,5 +1,14 @@
-<!-- column summary charts below -->
 <template>
+	<!-- Toggle histograms & column summary charts -->
+	<div class="datatable-toolbar">
+		<span class="datatable-toolbar-item"
+			>{{ csvHeaders?.length || 'No' }} columns | {{ csvContent?.length || 'No' }} rows</span
+		>
+		<span class="datatable-toolbar-item"
+			>Show column summaries<InputSwitch v-model="showSummaries"
+		/></span>
+	</div>
```

Comment:
The formatting is a bit weird here ... is this from your editor?

```
	<div class="datatable-toolbar">
		<span class="datatable-toolbar-item">
			{{ csvHeaders?.length || 'No' }} columns | {{ csvContent?.length || 'No' }} rows
		</span>
		<span class="datatable-toolbar-item">
			Show column summaries<InputSwitch v-model="showSummaries" />
		</span>
	</div>

```

Reply:
Prettier as tendancies to do this some times. It's not much of an issue as the next person editing the file will get that fixed automatically

Reply:
what's weird about it?